test/dicon/executor_test.exs

```diff
@@ -0,0 +1,12 @@
+defmodule Dicon.ExecutorTest do
```

Comment:
I think for `ExecutorTest` we need to define its own executor and don't modify the one in `test_helper.exs` (I think we can provide it as `Dicon.Test` in the future).